integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -37,8 +37,8 @@ spec:
         heritage: {{ .Release.Service }}
         role: alluxio-fuse
     spec:
-      hostNetwork: true
-      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: {{ .Values.fuse.hostNetwork }}
+      dnsPolicy: {{ .Values.fuse.dnsPolicy }}
```

Comment:
Parameterized these two fields as we do have users that use different setting for these fields. Same applies to all other resources.

Reply:
sgtm